README.md

```diff
@@ -112,10 +112,10 @@ Finally, properties are what give the selected elements of a rule declaration th
 ### Comments
 
 * Prefer line comments (`//` in Sass-land) to block comments.
-* Prefer comments on their own line. Avoid end-of-line comments.
```

Comment:
i thought we do prefer comments on their own line?

Reply:
I've mostly seen a mixture across our stylesheets :/
I was mostly following what I've seen rather than prescribing anything just yet. Do you think we should propose no same-line comments?
![image](https://user-images.githubusercontent.com/108884/39323750-82d8b65a-495b-11e8-8b93-0ba12302f641.png)

Reply:
yeah i think its nicer and it keeps line lengths down

Reply:
I'm gonna merge this in but we should definitely follow-up about comment styles